js/dropdowns.js

```diff
@@ -95,8 +99,8 @@ Dropdown.prototype.removePanel = function() {
 };
 
 Dropdown.prototype.checkIfEmpty = function() {
-  if (this.$panel.find('.dropdown__item').length === 0) {
-    this.removePanel();
+  if (this.$panel.find('input').length === 0) {
```

Comment:
This can be simplified to `return this.$panel.find('input').length === 0;`. I'll make this change on merging.